src/applications/keyboard/views/style.css

```diff
@@ -0,0 +1,3 @@
+.hg-button.hg-highlight {
+    box-shadow: 0 0 40px 5px rgba(64, 150, 255, 0.696);
```

Comment:
That's an oddly specific value for opacity 🤔

Reply:
Just used the color selector from VSCode, it's more or less random 😄